src/test/resources/examples/okHttpClient/client/LoggingInterceptor.kt

```diff
@@ -0,0 +1,25 @@
+package examples.okHttpClient.client
+
+import okhttp3.Interceptor
+import okhttp3.Response
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+class LoggingInterceptor : Interceptor {
+
+    private val logger: Logger = LoggerFactory.getLogger("GeneratedClient")
+
+    override fun intercept(chain: Interceptor.Chain): Response {
```

Comment:
I feel like this should be a configuration option. Forcing request and response logging on the user seems like a security risk